src/test/java/org/apache/mesos/testutils/NetworkTestUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.testutils;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+/**
+ * This class provides utilities useful for all tests which may have Network related needs.
+ */
+public class NetworkTestUtils {
+    public static int getRandomPort() throws IOException {
```

Comment:
Why throw IOException that's thrown by `serverSocket.close()` ? Just silence it, you already got your port.